.github/workflows/main.yml

```diff
@@ -69,9 +69,7 @@ jobs:
           runtime/spidermonkey/${{ matrix.profile }}
         key: cache-${{ hashFiles(
             'runtime/spidermonkey/build-engine.sh',
-            'runtime/spidermonkey/gecko-revision',
-            'runtime/spidermonkey/object-files.list',
-            'runtime/rust-toolchain.toml'
+            'runtime/spidermonkey/object-files.list'
```

Comment:
Won't this use cached builds for commits that change the gecko-dev submodule?

Reply:
silly me, yes you are correct, shall we add the entire gecko-dev folder?

Reply:
Would that consider the submodule hash, or the contents of gecko-dev? I think the submodule hash is what we want, so maybe that just works out :)

Reply:
I have added the commit hash of gecko-dev into the cache key [`a1dab90` (#636)](https://github.com/fastly/js-compute-runtime/pull/636/commits/a1dab90528f175617c02bb0964fd6fca79677295)

Reply:
Oh, you did that in your last commit already, thanks!